ase/sw/ase_common.h

```diff
@@ -441,15 +443,30 @@ bool remove_existing_lock_file(char *);
 #ifdef ASE_DEBUG
 void ase_buffer_info(struct buffer_t *);
 #endif
-void ase_buffer_oneline(struct buffer_t *);
 void ase_buffer_t_to_str(struct buffer_t *, char *);
 void ase_str_to_buffer_t(char *, struct buffer_t *);
 int ase_dump_to_file(struct buffer_t *, char *);
 uint64_t ase_rand64(void);
 void ase_eval_session_directory(void);
 int ase_instance_running(void);
-void remove_spaces(char *);
-void remove_tabs(char *);
+
+#ifdef __cplusplus
```

Comment:
shouldnt't the 'extern c_plusplus wrap all the functions in this file? 

i dont see any function on this file which it is implemented in a C++ implementation file..

Reply:
yes i understand the reason but can we enclose the complete set of function prototypes in this file in 'extern c_pluspus' since i think all of these functions are implemented in C, no one is implemented with C++